### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by aryszka**

### CODE REVIEW

**Comment by grassator** [comment]
👍

File: packaging/readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Packaging
+
+This directory may contain additional scripts and artefacts for packaging Skipper for different platforms or
+package managers.
+
+## Docker
```
Comment (danpersa) [inline packaging/readme.md]: will you push the Docker image to a repo?

Comment (aryszka) [inline packaging/readme.md]: yes

Comment (aryszka) [inline packaging/readme.md]: done

**Pull request merged**